pcdsdevices/epics_motor.py

```diff
@@ -56,15 +56,57 @@ class EpicsMotorInterface(FltMvInterface, EpicsMotor):
     disabled = Cpt(EpicsSignal, ".DISP", kind='omitted')
     # Description is valuable
     description = Cpt(EpicsSignal, '.DESC', kind='normal')
+    # Current Dial position
+    dial_position = Cpt(EpicsSignalRO, '.DRBV', kind='normal')
 
     tab_whitelist = ["set_current_position", "home", "velocity",
-                     "enable", "disable"]
+                     "enable", "disable", "check_limit_switches"]
 
     def __init__(self, *args, **kwargs):
         # Locally defined limit overrides, note can only make limits narrower
         self._limits = (-math.inf, math.inf)
         super().__init__(*args, **kwargs)
 
+    def format_status_info(self, status_info):
+        """
+        Override status info handler to render the motor.
+
+        Display motor status info in the ipython terminal.
+
+        Parameters
+        ----------
+        status_info: dict
+            Nested dictionary. Each level has keys name, kind, and is_device.
+            If is_device is True, subdevice dictionaries may follow. Otherwise,
+            the only other key in the dictionary will be value.
+
+        Returns
+        -------
+        status: str
+            Formatted string with all relevant status information.
+        """
+        # return 'N/A' if can't get the values for desired keys
+        description = status_info.get('description', {}).get('value', 'N/A')
+        units = status_info.get('user_setpoint', {}).get('units', 'N/A')
+        dial = status_info.get('dial_position', {}).get('value', 'N/A')
+        user = status_info.get('position', 'N/A')
+
+        low, high = self.limits
+        switch_limits = self.check_limit_switches()
+
+        name = ' '.join(self.prefix.split(':'))
+        name = f'{name}: {self.prefix}'
+        if description:
+            name = f'{description}: {self.prefix}'
+
+        return f"""\
+{name}
+Current position (user, dial): {user}, {dial} [{units}]
+User limits (low, high): {low}, {high} [{units}]
+Preset position: {self.presets.name}
```

Comment:
I think this isn't the right string for the preset position- I think it's `self.presets.state()`, referring to https://github.com/pcdshub/pcdsdevices/blob/5c1154159f3034a112b92044b1cbd44ba77c0e33/pcdsdevices/interface.py#L1187

Reply:
thank you @ZLLentz , this would make more sense than the `name` - I need to go back to all the PRs regarding status prints and make sure I'm printing the correct data

Reply:
I'm trying to check it closely during review: more eyes is better